Content.Server/Station/Systems/StationSpawningSystem.cs

```diff
@@ -106,12 +113,31 @@ private void OnStationInitialized(StationInitializedEvent ev)
             return jobEntity;
         }
 
-        if (!_prototypeManager.TryIndex(profile?.Species ?? HumanoidAppearanceSystem.DefaultSpecies, out SpeciesPrototype? species))
+        string speciesId;
+        if (_randomizeCharacters)
+        {
+            var weightId = _configurationManager.GetCVar(CCVars.ICRandomSpeciesWeights);
+            var weights = _prototypeManager.Index<WeightedRandomPrototype>(weightId);
+            speciesId = weights.Pick(_random);
```

Comment:
can the weights get used for the randomized humanoid appearance comp or w/e? if we tweak those enough we can add it back for nukies (unless they currently can still spawn as dionae and stuff i dont remember)

Reply:
nukies currently just use the player's preferred species (which i believe to be the best option)

Reply:
oh well